tests/unit/NavigationModelTest.php

```diff
@@ -181,102 +191,102 @@ function testParseMessage() {
 	function testParseOneLine() {
 		$model = new NavigationModel();
 
-		$cases = array();
+		$cases = [];
 
-		$cases[] = array(
+		$cases[] = [
 			'line' => "*whatever",
-			'out' => array(
+			'out' => [
 				'original' => 'whatever',
 				'text' => 'whatever',
-				'href' => Title::newFromText('whatever')->fixSpecialName()->getLocalURL(),
+				'href' => Title::newFromText( 'whatever' )->fixSpecialName()->getLocalURL(),
 				'specialAttr' => null,
-			)
-		);
+			]
+		];
 
-		$cases[] = array(
+		$cases[] = [
 			'line' => "*********whatever",
-			'out' => array(
+			'out' => [
 				'original' => 'whatever',
 				'text' => 'whatever',
-				'href' => Title::newFromText('whatever')->fixSpecialName()->getLocalURL(),
+				'href' => Title::newFromText( 'whatever' )->fixSpecialName()->getLocalURL(),
 				'specialAttr' => null,
-			)
-		);
+			]
+		];
 
-		$cases[] = array(
+		$cases[] = [
 			'line' => "*whatever|something",
-			'out' => array(
+			'out' => [
 				'original' => 'whatever',
 				'text' => 'something',
-				'href' => Title::newFromText('whatever')->fixSpecialName()->getLocalURL(),
+				'href' => Title::newFromText( 'whatever' )->fixSpecialName()->getLocalURL(),
 				'specialAttr' => null,
-			)
-		);
+			]
+		];
 
-		$cases[] = array(
+		$cases[] = [
 			'line' => "*http://www.google.com",
-			'out' => array(
+			'out' => [
 				'original' => 'http://www.google.com',
 				'text' => 'http://www.google.com',
 				'href' => 'http://www.google.com',
 				'specialAttr' => null,
-			)
-		);
+			]
+		];
 
-		$cases[] = array(
+		$cases[] = [
 			'line' => "*http://www.google.com|Google",
-			'out' => array(
+			'out' => [
 				'original' => 'http://www.google.com',
 				'text' => 'Google',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Google" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)